pages/record/_.vue

```diff
@@ -66,13 +102,40 @@
     },
     data() {
       return {
+        altTitle: null,
+        description: null,
         error: null,
         image: null,
         fields: null,
-        media: null
+        media: null,
+        title: null,
+        edmIsShownBy: {}
       };
     },
     computed: {
+      titlesInCurrentLanguage() {
```

Comment:
I suggest having two computed properties, one for mainTitle and one for altTitle. There may be a little duplication of logic, but it improves readability.

Reply:
With the new insights, as we discussed today, I kept one computed property for titles.